storage/store.go

```diff
@@ -1587,9 +1581,8 @@ func (s *Store) GetStatus() (*StoreStatus, error) {
 func (s *Store) computeReplicationStatus(now int64) (
 	leaderRangeCount, replicatedRangeCount, availableRangeCount int32) {
 	// Load the system config.
-	cfg, err := s.Gossip().GetSystemConfig()
-	if err != nil {
-		log.Error(err)
+	cfg := s.Gossip().GetSystemConfig()
+	if cfg == nil {
```

Comment:
picking nits now: i'd log at V(1) or higher, whatever the going rate in this file is